packages/portalnetwork/src/networks/network.ts

```diff
@@ -64,6 +64,10 @@ export abstract class BaseNetwork extends EventEmitter {
   public bridge: boolean
 
   portal: PortalNetwork
+  private lastRefreshTime: number = 0
+  private nextRefreshTimeout: ReturnType<typeof setTimeout> | null = null
+  private refreshInterval: number = 30000 // Start with 30s
```

Comment:
Is this used anywhere?

Reply:
Yes.  look into `bucketRefresh` and the new "interval" methods at the bottom of this file

Reply:
I think you can remove this line since you have the refresh interval options coded in the `getRefreshInterval` method at the bottom.